src/Database/Schema/PostgresSchema.php

```diff
@@ -393,6 +394,10 @@ public function columnSql(TableSchema $schema, string $name): string
             $out .= ' BYTEA';
         }
 
+        if ($data['type'] === TableSchema::TYPE_CHAR) {
+            $out .= '(' . (int)$data['length'] . ')';
```

Comment:
Why cast to int when doing string concatenation?

Reply:
A condition below this has cast it to (int). I found it weird too.. so for the sake of consistency.

Reply:
Let's try removing the casting and see if any tests fail :)

Reply:
Alrighty!